docs/source/notebooks/pcca.ipynb

```diff
@@ -96,29 +177,136 @@
     "pcca.coarse_grained_transition_matrix"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The membership probabilities can be accessed via `pcca.memberships`, where each column corresponds to a metastable set defining a probability distribution over the microstates:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "crisp_memberships = pcca.memberships.argmax(axis=1)\n",
+    "print(f\"Memberships: {pcca.memberships.shape}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(1, 2, figsize=(15, 10))\n",
+    "\n",
+    "for i in range(len(axes)):\n",
+    "    ax = axes[i]\n",
+    "    ax.set_title(f\"Metastable set {i+1} assignment probabilities\")\n",
+    "    \n",
+    "    ax.scatter(*data.T, c=pcca.memberships[assignments, i], cmap=plt.cm.Blues)\n",
+    "norm = mpl.colors.Normalize(vmin=0, vmax=1)\n",
+    "fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=plt.cm.Blues), ax=axes, shrink=.8);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, the assignments nicely disentangle the two ellipsoids.\n",
     "\n",
-    "f, ax = plt.subplots(1, 1, figsize=(12, 12))\n",
-    "ax.scatter(*data.T, c=assignments)\n",
-    "ax.scatter(*clustering.cluster_centers.T, marker=\"*\", \n",
-    "            c=['red' if m == 0 else 'cyan' for m in crisp_memberships])"
+    "One as also access to a coarse-grained stationary probability vector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pcca.coarse_grained_stationary_probability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another offered quantity are the metastable distributions, i.e., the probability of metastable states to visit a micro state by PCCA+ $\\mathbb{P}(\\text{state}_{\\text{micro}} | \\text{state}_\\text{pcca})$:"
```

Comment:
i'm not convinced this plot is the best way to visualize this (although i currently have no better ideas), but it's awesome to showcase/point out this quantity

Reply:
dito, lmk if you have ideas

Reply:
I think it looks better with the sorting!